frontend/express/public/localization/dashboard/dashboard.properties

```diff
@@ -516,11 +523,10 @@ report-manager.desc = Description
 report-manager.data = Data
 report-manager.period = Period
 report-manager.visibility = Visibility
-report-maanger.manually-created = Manually Created
-report-maanger.automatically-created = Automatically created
-report-maanger.manually-created-title = Manually Created Reports
-report-maanger.automatically-created-title = Automatically created Reports
-
+report-manager.manually-created = Manually Created
+report-manager.automatically-created = Automatically created
+report-manager.manually-created-title = Manually Created Reports
+report-manager.automatically-created-title = Automatically created Reports
```

Comment:
if you want to fix the correct localization string naming, you also need to fix the places where they are used in the code :)